electrum/plugins/revealer/qt.py

```diff
@@ -4,8 +4,9 @@
 Do you have something to hide?
 Secret backup plug-in for the electrum wallet.
 
-Tiago Romagnani Silveira, 2017
-
+Copyright:
+    2017 Tiago Romagnani Silveira
+    2023 Soren Stoutner <soren@stoutner.com>
```

Comment:
Now that you've also added the word "copyright", I would feel better if we explicitly specified the license.
Before your change, I would say it was implicitly MIT, inherited from the root directory license.

You can just add e.g.
```
Distributed under the MIT software license, see the accompanying
file LICENCE or http://www.opensource.org/licenses/mit-license.php
```
Or I can do it too if you like, just comment that it's ok :)

Reply:
That's a good Idea.  I'm a big fan of all files explicitly stating both their copyright and their license information.